Descent3/multi.h

```diff
@@ -784,9 +787,7 @@ extern int Num_network_games_known;
 extern int Game_is_master_tracker_game;
 
 #define TRACKER_ID_LEN 10 // Don't change this!
-extern char Tracker_id[TRACKER_ID_LEN];
-
-extern uint16_t Turrett_position_counter[MAX_OBJECTS];
```

Comment:
did `Turrett_position_counter` move or was it simply unused externally?

Reply:
`Turrett_position_counter` is not defined or even referenced anywhere in the code. Removal seemed like the right move.